src/redis-cli.c

```diff
@@ -2109,97 +2161,85 @@ static void repl(void) {
             char *endptr = NULL;
 
             argv = cliSplitArgs(line,&argc);
+            if (argv == NULL) {
+                printf("Invalid argument(s)\n");
+                fflush(stdout);
+                linenoiseFree(line);
```

Comment:
@huangzhw So unless you disagree, I guess we should also log the history here since we don't know if it's dangerous or not.

Reply:
It's OK. But blank line should not log?

Reply:
I would imagine no one would expect an empty line to log, the only time I can see that is just someone tapping enter a couple of times to move the cursor down.